CHANGELOG.md

```diff
@@ -23,7 +31,7 @@ All notable changes to this project will be documented in this file. The format
 
 ## [v6.3.0](https://github.com/puppetlabs/puppetlabs-apache/tree/v6.3.0) (2021-06-22)
 
-[Full Changelog](https://github.com/puppetlabs/puppetlabs-apache/compare/v6.2.0...v6.3.0)
+[Full Changelog](https://github.com/puppetlabs/puppetlabs-apache/compare/kps_ssl_reload_and_cache_disk_combined_tag...v6.3.0)
```

Comment:
This looks weird. I wonder who pushed this tag. Should we remove it?

Reply:
Thought we already had? Removed it again and rerunning release.